sensorserviceutils/sensorserviceutils.go

```diff
@@ -173,6 +173,11 @@ type parser struct {
 	parsingErrs []error
 	sensorErrs  []error
 
+	// apps is a map from UID to a sensorcheck map.
```

Comment:
sensorcheck - what do you mean by that? Do you mean
apps is a map from UID to active sensors.

Reply:
I think I will rephrase this comment.
apps is a map from uid to another map that indicates whether the app with this UID has a subscription event for the sensor. 
For example, if the inner map for UID=100 maps SensorNumber 3 to true,(p.apps[100][3] = true), it means that app with UID 100 has subscribed sensor number 3 in the history.